gluster-exporter/metric_brick.go

```diff
@@ -451,6 +619,18 @@ func brickUtilization() error {
 						// Convert to bytes
 						glusterBrickLVMetadataSize.With(lvmLbls).Set(stat.MetadataSize * 1024 * 1024)
 						glusterBrickLVMetadataPercent.With(lvmLbls).Set(stat.MetadataPercent)
+						glusterVGExtentTotal.With(lvmLbls).Set(stat.VGExtentTotal)
+						glusterVGExtentAlloc.With(lvmLbls).Set(stat.VGExtentAlloc)
+						glusterVGExtentFree.With(lvmLbls).Set(stat.VGExtentFree)
+					}
+					for _, thinStat := range thinStats {
+						var thinLvmLbls = getGlusterThinPoolLabels(brick, subvol.Name, thinStat)
+						glusterThinPoolDataTotal.With(thinLvmLbls).Set(thinStat.ThinPoolDataTotal * 1024 * 1024)
+						glusterThinPoolDataUsed.With(thinLvmLbls).Set(thinStat.ThinPoolDataUsed * 1024 * 1024)
+						glusterThinPoolDataFree.With(thinLvmLbls).Set(thinStat.ThinPoolDataFree * 1024 * 1024)
+						glusterThinPoolMetadataTotal.With(thinLvmLbls).Set(thinStat.ThinPoolMetadataTotal * 1024 * 1024)
+						glusterThinPoolMetadataUsed.With(thinLvmLbls).Set(thinStat.ThinPoolMetadataUsed * 1024 * 1024)
+						glusterThinPoolMetadataFree.With(thinLvmLbls).Set(thinStat.ThinPoolMetadataFree * 1024 * 1024)
```

Comment:
can free be determined from total and used? do we need the additional metric?

Reply:
Yes thats how its found. I noticed the brick capacity metrics had used, free and total, so thought thats the standard. If free is to be monitored then the user can get the derived metric from used and total as a recording rule in prometheus. So i'll remove it?

Reply:
Yes, you can remove that and rule in prometheus can derive that value

Reply:
Ack